memphis/memphis.py

```diff
@@ -46,9 +46,12 @@ def cancel(self):
 
 
 class Memphis:
-
     def __init__(self):
         self.is_connection_active = False
+        self.t_schema_updates = None
+        self.producers_count = 0
```

Comment:
this counter should be made for producers per specific station and not total producers to all stations under this connection

Reply:
I did it in the map of schema data updates and save the producers counts per station . I saved the producers count in the map of schema data updates instead of creating one more map. 

Now I separated  the producers count per station name to new map.